routes/api.php

```diff
@@ -18,4 +18,8 @@
 });
 
 Route::get('articles', 'Api\Article\ArticleApiController@index');
-Route::post('articles', 'Api\Article\ArticleApiController@store');
\ No newline at end of file
+Route::post('articles', 'Api\Article\ArticleApiController@store');
```

Comment:
Instead of declaring all the routes. Use [Resource Route](https://laravel.com/docs/6.0/controllers#restful-partial-resource-routes)